src/kms/kms.ts

```diff
@@ -801,6 +856,65 @@ export class KmsClient {
     )
     return publicMasterKey.policy()
   }
+
+  /**
+   * Get and wrap
+   * @param uniqueIdentifier the unique identifier of the object to get and wrap
+   * @param encryptionKeyUniqueIdentifier the unique identifier to use to wrap the fetched key
+   * @returns wrapped object
+   */
+  public async getWrappedKey(
```

Comment:
Do you think we could merge this function in the `getObject` function?
If we want to ease the use/creation of the `KeyWrappingSpecification` object, we could replace (in the actual `getObject` function) the `KeyWrappingSpecification` parameter by `encryptionKeyUniqueIdentifier` and let the `getObject` function recreate locally the `KeyWrappingSpecification`.